packages/modules/web_themes/standard/source/src/stores/mqtt-store.ts

```diff
@@ -1883,10 +1948,12 @@ export const useMqttStore = defineStore('mqtt', () => {
     chargePointConnectedVehicleChargeTemplate,
     // vehicle data
     vehicleList,
+    chargePointConnectedVehicleSoc,
     vehicleAddScheduledChargingPlan,
     vehicleDeleteScheduledChargingPlan,
     vehicleScheduledChargingPlans,
     vehicleToggleScheduledChargingPlanActive,
```

Comment:
```suggestion
    vehicleScheduledChargingPlanActive,
```
Bitte auch die Verweise in anderen Dateien prüfen.

Reply:
Changed